extensions-core/multi-stage-query/src/test/java/org/apache/druid/msq/sql/resources/SqlStatementResourceTest.java

```diff
@@ -935,14 +934,45 @@ public void testForbiddenRequest()
         ).getStatus()
     );
     Assert.assertEquals(
-        Response.Status.FORBIDDEN.getStatusCode(),
+        Response.Status.ACCEPTED.getStatusCode(),
         resource.deleteQuery(
             RUNNING_SELECT_MSQ_QUERY,
             makeExpectedReq(CalciteTests.SUPER_USER_AUTH_RESULT)
         ).getStatus()
     );
   }
 
+  @Test
+  public void testAPIBehaviourWithForbiddenUser()
```

Comment:
We should add the following test cases: 

And one test case where the user has state READ perms ONLY 
And one test case where the user has state WRITE perms only 
And one test case where the user has both READ and WRITE. 

In all cases the query is submitted by a different user.